vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageAudioItem.kt

```diff
@@ -37,8 +37,8 @@ import im.vector.app.features.home.room.detail.timeline.helper.ContentUploadStat
 import im.vector.app.features.home.room.detail.timeline.style.TimelineMessageLayout
 import im.vector.app.features.themes.ThemeUtils
 
-@EpoxyModelClass(layout = R.layout.item_timeline_event_base)
-abstract class MessageAudioItem : AbsMessageItem<MessageAudioItem.Holder>() {
+@EpoxyModelClass
+abstract class MessageAudioItem : AbsMessageItem<MessageAudioItem.Holder>(R.layout.item_timeline_event_base) {
```

Comment:
Maybe not a good idea, but `AbsMessageItem` may have `R.layout.item_timeline_event_base` as a default value for `layoutId`, WDYT?

Reply:
make sense, I didn't realise they were all using the exact same base layout https://github.com/vector-im/element-android/pull/6415/commits/d0e5b3eb21232abcc87d10100cc9a9e90c14bf5a